homeassistant/components/mqtt/client.py

```diff
@@ -392,7 +392,8 @@ async def async_publish(
                 self._mqttc.publish, topic, payload, qos, retain
             )
             _LOGGER.debug(
-                "Transmitting message on %s: '%s', mid: %s",
+                "Transmitting%s message on %s: '%s', mid: %s",
```

Comment:
```suggestion
                "Transmitting %s message on %s: '%s', mid: %s",
```
Does `Transmitting` and the `topic` without a space by purpose?

Reply:
The first `%s` is substituted with either `""` or `" retained"`, your suggested change introduces an unwanted space. Same for the 2nd suggestion.

Reply:
Looking at it again it produce the intended result 👍